Mopups/Mopups.Maui/Platforms/Windows/Impl/PopupPlatformWindows.cs

```diff
@@ -51,7 +51,8 @@ public static bool SendBackPressed(Action? backPressedHandler = null)
 
         public async Task AddAsync(PopupPage page)
         {
-            page.Parent = Application.Current.MainPage;
+            var mainPage = Application.Current.MainPage;

```

Comment:
By the way, here's a comment for the code maintainers (@LuckyDucko): Using Application.Current.MainPage is deprecated and will not work when there are multiple windows. The AddAsync method should be given some context so it can find the right window.

For example, here's how MAUI Community Toolkit does it (it has a Page context):

* https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/popup#presenting-a-popup